bagri-server/bagri-server-hazelcast/src/main/java/com/bagri/server/hazelcast/impl/DocumentManagementImpl.java

```diff
@@ -1395,27 +1317,6 @@ public int removeDocumentFromCollections(String uri, String[] collections) {
 		logger.trace("removeDocumentsFromCollections.exit; removed: {}; unknown: {}", remCount, unkCount);
 		return remCount;
 	}
-	
-	private boolean lockDocument(DocumentKey docKey, long timeout) { //throws XDMException {

```

Comment:
the lock/unlock methods were planed to add into DocManagement API.

Reply:
returned back